node/mock-task.ts

```diff
@@ -64,6 +62,32 @@ module.exports.getBoolInput = task.getBoolInput;
 module.exports.getDelimitedInput = task.getDelimitedInput;
 module.exports.filePathSupplied = task.filePathSupplied;
 
+export function getVariable(name: string): string | undefined  {
+    const variables = mock.getVariableMap();
+
+    if (variables !== undefined) {
+        if (variables[name]) {
```

Comment:
Just checking, does this support mocking a variable to be undefined?

Reply:
You can't explicitly state that variable is undefined. The absence of a variable in the answers collection makes it undefined. I believe ultimately the real `getVariables` only returns defined variables.

Related, if I run `process.env["ZZZ"] = undefined` in node, `process.env["ZZZ"]` actually gets set to string "undefined". `for (const name in process.env) { if (process.env[name] === undefined) { console.log(name) } }` returns no entries.

Reply:
`getVariable`'s behavior is hopefully a little clearer now.